libs/gltfio/src/AssetLoader.cpp

```diff
@@ -147,10 +157,11 @@ struct FAssetLoader : public AssetLoader {
 
     EntityManager& mEntityManager;
     RenderableManager& mRenderableManager;
-    NameComponentManager* mNameManager;
+    NameComponentManager* const mNameManager;
```

Comment:
Why a pointer and not a `&` just like the ones above?

Reply:
Fixed.